README.md

```diff
@@ -46,7 +46,6 @@ We include the following models:
 - Hold
 - Recurring Schedule
 - Schedule Eraser
-- Slot
```

Comment:
We're adding this back in the other PR:

- https://github.com/tipoff/scheduler/pull/45